ext_emconf.php

```diff
@@ -31,7 +31,7 @@
     'constraints' => [
         'depends' => [
             'php' => '7.0.0-7.3.99',
-            'typo3' => '8.7.0-9.5.99',
+            'typo3' => '9.5.0-9.5.99',
```

Comment:
Perfect. No TYPO3 8 compatibility anymore.

Reply:
With nullable type introduced above, dependency for `php` needs to be adapted as well, and if this targets v9, then PHP 7.1 is not supported either. `composer.json` needs to be adapted as well.

Reply:
What about rising PHP range to 7.1.0.-7.4.99?

Reply:
done

Reply:
TYPO3 9 needs at least PHP 7.2. But I prefer to remove this php dependency completely s TYPO3 checks PHP dependency in Bootstrap class.